repos/spack_repo/builtin/packages/geant4_vmc/package.py

```diff
@@ -36,10 +38,12 @@ class Geant4Vmc(CMakePackage):
     version("4-0-p2", sha256="cdd73c499cd296f13b6c0d37e161e7d94343f85617b2a7577ded8312248f9b9b")
     version("3-6-p6", sha256="e62a62ff7075ff9afb2ffe420610374f62136094a447bbbc5f739a2238ddb0f0")
 
+    depends_on("c", type="build")
     depends_on("cxx", type="build")  # generated
 
     depends_on("cmake@3.3:", type="build")
```

Comment:
Noticed that a newer version of CMake is required but did not verify that v6-7 is when it changed:
```suggestion
    depends_on("cmake@3.3:", type="build")
    depends_on("cmake@3.16:", type="build", when="@6-7:")
```